docs/docs/troubleshooting.mdx

```diff
@@ -200,3 +221,41 @@ First, you will need to create new node groups, one for each type of GPU instanc
 
 Second, you will need to create a new JupyterLab profile to select the GPU node.
 In this way, you'll have a separate profile for GPUs which users can select.
+
+### Migrate existing environments from a namespaces to another
+
+When the user creates a new environment, it requires selecting a namespace. This namespace is used to identify the group permissions to
+that environment and the location where the such environment needs to be stored.
+To migrate environments from one namespace to another, the user must manually build all environments in the new namespace.
```

Comment:
```suggestion
To migrate environments from one namespace to another, you must manually build all environments in the new namespace.
```

Reply:
@viniciusdc A quick question so that I understand this better, do just the administrators (i.e., people managing the qhub/nebari deployment and user groups) need to do this migration, or do all the people (for example, I'm thinking data scientists using JupyterLab) using the platform?

Reply:
That depends a lot on the excatly situation:
- If the user (data scientist) has permission over the namespace (e.g. its currently assigned group matches the target namespace), then its possible for them to do these changes
- Admins normally have access to all namespaces, so this extra validation is not needed
- There is only the caveat of the Nebari system managed namespaces `nebari-git` (currently `filesystem`), and `global` (`default`), wich normally are completely restricted to anyone -- to change this a Cluster administrator needs to intervene

Reply:
Got it, thanks!